client/state/action-types.ts

```diff
@@ -617,6 +617,7 @@ export const PLUGIN_UPDATE_REQUEST = 'PLUGINS_UPDATE_REQUEST';
 export const PLUGIN_UPDATE_REQUEST_FAILURE = 'PLUGIN_UPDATE_REQUEST_FAILURE';
 export const PLUGIN_UPDATE_REQUEST_SUCCESS = 'PLUGIN_UPDATE_REQUEST_SUCCESS';
 export const PLUGIN_ALREADY_UP_TO_DATE = 'PLUGIN_ALREADY_UP_TO_DATE';
+export const PLUGIN_ACTION_STATUS_UPDATE = 'PLUGIN_ACTION_STATUS_UPDATE';
```

Comment:
Is this intentional?

Reply:
It was out of sync from Trunk. Now it's good.